internal/core/thirdparty/knowhere/CMakeLists.txt

```diff
@@ -21,13 +21,19 @@ else ()
         "https://github.com/milvus-io/knowhere/archive/refs/tags/${KNOWHERE_VERSION}.tar.gz" )
 endif ()
 
+if (MSYS)
+    # windows adaptation + milvus-io/knowhere@v1.0.1
+    set (KNOWHERE_SOURCE_URL "https://github.com/matrixji/knowhere/archive/refs/tags/windows-int-20220227.tar.gz")
+    set (KNOWHERE_SOURCE_MD5 "3b3a911c4647804b42cf8b7d8d696b1d")
+endif (MSYS)
+
```

Comment:
Please add a TODO comment, so that as soon as the knowhere windows patch is merged, we're able to switch to the official knowhere.

Reply:
Fixed in dd07fab5e